docs/advanced_usage/step_report.rst

```diff
@@ -1,3 +1,5 @@
+.. _step_test:
+
```

Comment:
Why is the title system-test report? Can't it be generated for any test? 
I'd go for something more generic like `HTML step-report generation`